test/test_benchmark.py

```diff
@@ -552,7 +552,8 @@ def test_memops(self):
     '''
     self.do_benchmark('memops', src, 'final:')
 
-  def zzztest_files(self):
+  @non_core
+  def test_files(self):
```

Comment:
Does the `zzz` prefix disable things?  I though it just made them appear last in the alphabetically sorted list of tests.

Reply:
Oh that is confusing.  Do you think original intention here was to disable them or to put them last?    It seems that `@disabled` would have made more sense if the idea was to disable them.

Anyway lgtm!

Reply:
Should we rename to `test_zzz_...` or do you think it's not necessary?

Reply:
@sbc100 Not sure about the original intention. I guess @kripken might know more.
@tlively In this file, `test_zzz_...` is used for big benchmarks like bullet and poppler, to make them listed last. I don't think that applies here.

Reply:
the `zzz` prefix is used to push them to the end, yeah, since it is sorted. And we put the big/slow ones at the end. This one seems fine without `zzz`. But yeah, let's make it non-core.

Reply:
The problem is that in this case it was called `zzztest_files` instead of `test_zzz_files`, which meant the test was completely disabled, not just sorted to come last (test methods are requires to start with `test_`)